src/main/scala/io/flow/localization/rates/Localizer.scala

```diff
@@ -124,28 +125,48 @@ class LocalizerImpl @Inject() (dataClient: DataClient, rateProvider: RateProvide
     implicit executionContext: ExecutionContext
   ): Future[Option[FlowSkuPrice]] = {
     val key = CountryKey(country, itemNumber)
-    getPricing(key)
+    val defaultCurrency = Countries.mustFind(country).defaultCurrency
+    getPricing(key, defaultCurrency)
   }
 
   override def getSkuPricesByCountry(country: String, itemNumbers: Iterable[String])(
     implicit executionContext: ExecutionContext
   ): Future[List[Option[FlowSkuPrice]]] = {
     val keys = itemNumbers.map(CountryKey(country, _))
-    getPricings(keys)
+    val defaultCurrency = Countries.mustFind(country).defaultCurrency
```

Comment:
Should we define this case in the fallback case only (when the first keys have not been found) and instead of `mustFind` let's maybe log something and return `None`?

Reply:
refactored to only defined in the fallback cases.  the `mustFind` exists on the finding the `country` but we'll still return `None` on the finding the `defaultCurrency` itself